### PR TITLE
Add Blackmagic Desktop Video 10.9.7

### PR DESCRIPTION
**Pull request by NaanProphet**
Drivers for Blackmagic Design hardware, used in video capture/editing

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-drivers/pulls
[closed issues]: https://github.com/caskroom/homebrew-drivers/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by NaanProphet** [comment]
One snag at the moment in the uninstall stanza causes the GUI to popup. Not sure how to workaround that... Is that an problem?

**Comment by commitay** [comment]
Please submit this to [`caskroom/drivers`](https://github.com/caskroom/homebrew-drivers)

**Comment by NaanProphet** [comment]
Ah, meant to do that! My bad thanks.

File: Casks/blackmagic-desktop-video.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'blackmagic-desktop-video' do
+  version '10.9.7'
+  sha256 '84bd14be508332d2da010d43a7eae74af1305a48052b72aaa9708e8143b4d9ec'
+
+  url 'http://sw.blackmagicdesign.com/DesktopVideo/v10.9.7/Blackmagic_Desktop_Video_Macintosh_10.9.7.zip?Key-Pair-Id=APKAJTKA3ZJMJRQITVEA&Signature=I4/V92Wv8nowO5hI7TiITzmGjaPVVXbZdzAb9XXRkn1GwaXJ4IQ7KG3Cthym4AUzI+ZwWMkESCLyMoyfFWECmusjPY7+SPFPWlKW0vEKSr+CtfHqCFk4Nq1FJ6hcmqMAXB+I6DI2pG5iDGaj3AHCyVWHUZI7HGEIckElKWNnPyZWn6vOVcGD3GNi0jZwbmCB0uKhjrj6NOfYj6+3Z5NedtJFdvTU3NYJDaOocbs6KlKvE04peBDU3Z1psT5wiKKIPAi0YnXHIdxbX3XefYY1iOgZa7nA2DrUgb+lekNef9skwzR69h8CpKoU5Ad6Z9CO2nOM0mynZWiS8pHXbZ7AUw==&Expires=1506801058'
```
Comment (commitay) [inline Casks/blackmagic-desktop-video.rb]: This URL will expire, unless you can find a direct download we cannot accept this Cask.

**Comment by NaanProphet** [comment]
Actually wait--the link to caskroom-drivers takes me back here. Am i missing something?

**Comment by commitay** [comment]
> Actually wait--the link to caskroom-drivers takes me back here. Am i missing something?

Sorry, my mistake.

File: Casks/blackmagic-desktop-video.rb (hunk 1 above)
Comment (NaanProphet) [reply Casks/blackmagic-desktop-video.rb]: Ah, didn't see that! Thanks

**Comment by NaanProphet** [comment]
Rats, i can't really see a way around the key/pair download link... Thanks for the guidance though!

**Pull request closed without merging**